examples/python/pincell_multigroup/build-xml.py

```diff
@@ -0,0 +1,184 @@
+
+# Instantiate the energy group data
+groups = openmc.mgxs.EnergyGroups(group_edges=[1E-11, 0.0635E-6, 10.0E-6,
+                                               1.0E-4, 1.0E-3, 0.5, 1.0, 20.0])
+
+# Instantiate the 7-group (C5G7) cross section data
+uo2_xsdata = openmc.XSdata('UO2.300k', groups)
+uo2_xsdata.order = 0
+uo2_xsdata.total = np.array([0.1779492, 0.3298048, 0.4803882, 0.5543674,
+                            0.3118013, 0.3951678, 0.5644058])
+uo2_xsdata.absorption = np.array([8.0248E-03, 3.7174E-03, 2.6769E-02, 9.6236E-02,
+                                  3.0020E-02, 1.1126E-01, 2.8278E-01])
+scatter = [[[0.1275370, 0.0423780, 0.0000094, 0.0000000, 0.0000000, 0.0000000, 0.0000000],
+            [0.0000000, 0.3244560, 0.0016314, 0.0000000, 0.0000000, 0.0000000, 0.0000000],
+            [0.0000000, 0.0000000, 0.4509400, 0.0026792, 0.0000000, 0.0000000, 0.0000000],
+            [0.0000000, 0.0000000, 0.0000000, 0.4525650, 0.0055664, 0.0000000, 0.0000000],
+            [0.0000000, 0.0000000, 0.0000000, 0.0001253, 0.2714010, 0.0102550, 0.0000000],
+            [0.0000000, 0.0000000, 0.0000000, 0.0000000, 0.0012968, 0.2658020, 0.0168090],
+            [0.0000000, 0.0000000, 0.0000000, 0.0000000, 0.0000000, 0.0085458, 0.2730800]]]
+uo2_xsdata.scatter = np.array(scatter[:][:])
+uo2_xsdata.fission = np.array([7.21206E-03, 8.19301E-04, 6.45320E-03,
+                               1.85648E-02, 1.78084E-02, 8.30348E-02,
+                               2.16004E-01])
+uo2_xsdata.nu_fission = np.array([2.005998E-02, 2.027303E-03, 1.570599E-02,
+                                  4.518301E-02, 4.334208E-02, 2.020901E-01,
+                                  5.257105E-01])
+uo2_xsdata.chi = np.array([5.8791E-01, 4.1176E-01, 3.3906E-04, 1.1761E-07,
+                           0.0000E+00, 0.0000E+00, 0.0000E+00])
+
+h2o_xsdata = openmc.XSdata('LWTR.300k', groups)
+h2o_xsdata.order = 0
+h2o_xsdata.total = np.array([0.15920605, 0.412969593, 0.59030986, 0.58435,
+                             0.718, 1.2544497, 2.650379])
+h2o_xsdata.absorption = np.array([6.0105E-04, 1.5793E-05, 3.3716E-04,
+                                  1.9406E-03, 5.7416E-03, 1.5001E-02,
+                                  3.7239E-02])
+scatter = [[[0.0444777, 0.1134000, 0.0007235, 0.0000037, 0.0000001, 0.0000000, 0.0000000],
+            [0.0000000, 0.2823340, 0.1299400, 0.0006234, 0.0000480, 0.0000074, 0.0000010],
+            [0.0000000, 0.0000000, 0.3452560, 0.2245700, 0.0169990, 0.0026443, 0.0005034],
+            [0.0000000, 0.0000000, 0.0000000, 0.0910284, 0.4155100, 0.0637320, 0.0121390],
+            [0.0000000, 0.0000000, 0.0000000, 0.0000714, 0.1391380, 0.5118200, 0.0612290],
+            [0.0000000, 0.0000000, 0.0000000, 0.0000000, 0.0022157, 0.6999130, 0.5373200],
+            [0.0000000, 0.0000000, 0.0000000, 0.0000000, 0.0000000, 0.1324400, 2.4807000]]]
+h2o_xsdata.scatter = np.array(scatter[:][:])
```

Comment:
should have been that simple.  Fixed.  The outermost list is to represent the scattering order.